schema/types/isolator_linux_specific_test.go

```diff
@@ -94,3 +94,113 @@ func TestNewLinuxCapabilitiesRevokeSet(t *testing.T) {
 	}
 
 }
+
+func TestNewLinuxSeccompRemoveSet(t *testing.T) {
+	tests := []struct {
+		set   []string
+		errno string
+
+		wset   []LinuxSeccompEntry
+		werrno LinuxSeccompErrno
+		werr   bool
```

Comment:
To me it stands for "wanted" (as in got/want everywhere else), but I was just following the convention of the surrounding tests (so it may be something different).